src/App.js

```diff
@@ -1,25 +1,62 @@
 import React from 'react';
+import classNames from 'classnames';
+import Clock from './components/Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: Math.floor(Math.random() * 10),
+  };
+
+  componentDidMount() {
+    setInterval(() => {
+      const date = new Date();
+
+      if (this.state.isClockVisible) {
+      // eslint-disable-next-line
+      console.log(date.toLocaleTimeString());
+      }
+    }, 1000);
+  }
+
+  hideClock = () => {
+    this.setState({ isClockVisible: false });
+  }
+
+  showClock = () => {
+    this.setState({ isClockVisible: true });
+  }
+
+  render() {
+    const visibility = this.state.isClockVisible;
+    const hide = this.hideClock;
+    const show = this.showClock;
+
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        <p
+          className={classNames({ 'App__clock-hidden': !visibility })}
+        >
+          Current time:
+          {' '}
+          <span><Clock name={this.state.clockName} /></span>
```

Comment:
```suggestion
        {visibility && <Clock name={this.state.clockName} />}
```
You can also render or not render your clock component depending on a condition